### PR TITLE
Update 2012 head prefect's name

### LINKED ISSUE #143: Update Head prefect page
**Is your feature request related to a problem? Please describe.**

- Including the 2020 head prefects' name

**Describe the solution you'd like**

- Add the necessary code

**Are you working on it?**
yes

### PR DESCRIPTION
**Pull request by mushrifahamed**
## Purpose

The purpose of this PR is to fix #143 

## Goals
View 2020 Head prefects' name on the official website

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by mushrifahamed** [comment]
everything placed in order

File: head_prefects.html [hunk 1]
```diff
@@ -1,265 +1,267 @@
...
 				$("#footer").load("assets/content/footer.html");
 			});
-		</script>
-			
-	</body>
+
+</script>
+
+</body>
 </html>
```
Comment (Piumal1999) [inline head_prefects.html:267]: Keep a blank line at the end of the file

Comment (mushrifahamed) [reply head_prefects.html:267]: changes done

**Comment by mushrifahamed** [comment]
Requested changes done

**Comment by kumuditha-udayanga** [comment]
@all-contributors please add @mushrifahamed  for code

**Pull request merged**